README.md

```diff
@@ -7,6 +7,13 @@ TFS micro-service provides endpoints for internal trade finance manager systems
 npm install
 ```
 
+## Environment Variables
+A member of the team should send an encrypted copy of the `.env` file and password. To decrypt the file run the following command and enter the password when prompted.
+
+```bash
+$ winpty openssl enc -aes-256-cbc -pbkdf2 -d -in <path to encrypted file> -out <path to save decrypted file>
+```
```

Comment:
I think we should keep the first sentence, but the 2nd sentence and the command will depend on how the team member decides to encrypt their file so maybe we shouldn't specify it here